modules/web-console/web-console-server/src/main/resources/ignite-config.xml

```diff
@@ -45,8 +63,17 @@
             </bean>
         </property>
 
+        <property name="connectorConfiguration">
+            <bean class="org.apache.ignite.configuration.ConnectorConfiguration">
+                <property name="port" value="${random.int[10000,65000]}"/>
```

Comment:
Why to use random port? I think this can cause "strange" from user point error. Lets just use value that will differ from default one "11211", for example: "22122"?